include/seqan3/io/sequence_file/input.hpp

```diff
@@ -307,12 +305,7 @@ struct sequence_file_input_default_traits_aa : sequence_file_input_default_trait
 
 template <
     sequence_file_input_traits traits_type_ = sequence_file_input_default_traits_dna,
-    detail::fields_specialisation selected_field_ids_ = fields<field::seq, field::id, field::qual>,
-    detail::type_list_of_sequence_file_input_formats valid_formats_ = type_list<format_embl,
-                                                                                format_fasta,
-                                                                                format_fastq,
-                                                                                format_genbank,
-                                                                                format_sam>>
```

Comment:
For an easier upgrade-path, I would want to keep this template parameter for now. But effectively shutting that template parameter down. I'm not sure how to emit deprecation notices on template parameters, but we can think of a creative way :)

I have this rough idea, how to do it:
https://godbolt.org/z/fPq4sG

Reply:
Ok, then I will first apply the internal changes and keep the templates as are. Then this must be deprecated in a follow up PR.